pipeline/start_server.py

```diff
@@ -73,26 +75,36 @@
     )
     args = parser.parse_args()
 
-    args.model_path = Path(args.model_path).absolute()
+    if os.path.exists(args.model_path):
```

Comment:
Maybe better to make model_path optional argument and add another argument model_name for your usecase? And if it's provided, we still do all the checks below. Otherwise if we provide incorrect model path, there will be no error, until the model is actually loaded inside slurm job

Reply:
Hmm that's an option but it opens up too many combinations and we have to sit and reject half the combinations that don't work. I can make it more robust by saying model path split is length two and that the file does not exist, which should be sufficient to assert that it's a HF path

Reply:
why is that sufficient? You can currently specify relative model path as `trt_models/some_model_name`, which is what I do often. Feels safer to have an extra parameter and then just a check if both are specified or none are specified, we raise an error. Especially given that in some of the servers (e.g. trtllm), the error message when it can't find the model is super obscure, I think it just fails with segfault and you'd spend some time trying to figure out what happened while you simply had a path typo

Reply:
Oof ok makes sense

Reply:
Made the changes